includes/admin/models/class-wppr-review-model.php

```diff
@@ -195,6 +196,23 @@ public function __construct( $review_id = false ) {
 		return false;
 	}
 
+	/**
+	 * Setup hooks if this review is a CPT.
+	 */
+	private function setup_cpt() {
+		if ( 'wppr_review' === get_post_type( $this->ID ) ) {
+			add_filter( 'wppr_name', array( $this, 'get_name_for_cpt' ), 10, 2 );
+		}
+	}
+
+	/**
+	 * If this is a CPT, use the post title as the product name.
+	 */
+	public function get_name_for_cpt( $name, $id ) {
+		$post   = get_post( $id );
```

Comment:
can you use ? https://developer.wordpress.org/reference/functions/get_the_title/

Reply:
in get_the_title(), `If the post is protected and the visitor is not an admin, then "Protected" will be displayed before the post title`. Are we fine with this?

Reply:
yep, that's fine.

Reply:
@selul internally this method does the same thing we are doing so what would be the reason for preferring this?

Reply:
@selul done